code/TemplateStudioForWinUICs/Templates/Proj/Default/Param_ProjectName/MainWindow.xaml

```diff
@@ -10,7 +10,4 @@
     MinHeight="500"
     PersistenceId="MainWindow"
     mc:Ignorable="d">
-    <windowex:WindowEx.Backdrop>
```

Comment:
Can we keep this defined in XAML like https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.media.micabackdrop?view=windows-app-sdk-1.3#examples shows?

Reply:
Yes.

Reply:
Don't think so :/

![image](https://user-images.githubusercontent.com/40738104/234336659-77f53046-2cbe-40eb-bd80-b3c9d1f0f53b.png)
![image](https://user-images.githubusercontent.com/40738104/234336722-28af09ab-2861-44ab-8230-23453c1bc636.png)

Reply:
And with that

https://github.com/microsoft/microsoft-ui-xaml/pull/8139/files#diff-4f203bcadb6647af5b21c52a5d9464ca0c85fe096fd5cfd856626d53aa2e7882R99

Reply:
And when I try to remove MicaKind, I have this result

![image](https://user-images.githubusercontent.com/40738104/234338515-bce81e2c-9240-462c-9038-59b741ba1d33.png)
![image](https://user-images.githubusercontent.com/40738104/234338373-e70e97fe-8a75-4aea-8f78-c8fd55bedc31.png)

Reply:
@dotmorten have you tried this yet?

Reply:
He said that.

https://github.com/microsoft/TemplateStudio/issues/4641#issuecomment-1516902916

Reply:
Odd the doc shows this. It wasn’t working for me

Reply:
So, we wait a fix ?

Reply:
Yes. Looks like they're taking it through servicing so the fix should be released soon.

Reply:
Servicing update 1.3.1 contains this fix and is rolling out as we speak. The nuget package is already available.

Reply:
Nice 👍. Servicing update would be released before the end of the week?

Reply:
I update the version of WASDK. It's done.